app/views/image/_reuse_image.html.erb

```diff
@@ -12,18 +12,25 @@
       <p class="help-block"><%= :image_reuse_id_help.tp %></p>
 
       <div class="form-group mt-3">
-        <%= link_with_query(@all_users ? :image_reuse_just_yours.t : :image_reuse_all_users.t,
+        <%= link_with_query((@all_users ? :image_reuse_just_yours.t :
+                              :image_reuse_all_users.t),
                             url_args.merge(all_users: @all_users ? 0 : 1),
                             class: "btn btn-default") %>
       </div>
     </div>
   </div>
 
   <%= paginate_block(@pages) do %>
-    <%= render(layout: "shared/matrix_table", locals: {objects: @objects}) do |image| %>
-      <div class="mt-3 mb-3">
-        <%= thumbnail(image, link: url_args.merge(img_id: image.id), original: true) %>
-      </div>
-    <% end %>
+    <ul class="row list-unstyled mt-3">
+      <%= @objects.each_with_index do |image, idx| %>
+        <li class="matrix-box col-xs-12 col-sm-6 col-md-4">
+          <div class="panel panel-default">
+            <div class="panel-body">
+              <%= thumbnail(image, link: url_args.merge(img_id: image.id), original: true) %>
+            </div>
+          </div>
+        </li>
+      <% end %>
+    </ul>
```

Comment:
Why does this generate HTML with an `unsafe_html` error:
```ruby
      <%= @objects.each_with_index do |image, idx| %>
        <li class="matrix-box col-xs-12 col-sm-6 col-md-4">
          <div class="panel panel-default">
            <div class="panel-body">
              <%= thumbnail(image, link: url_args.merge(img_id: image.id), original: true) %>
            </div>
          </div>
        </li>
      <% end %>
```
But this does not?
```ruby
      <%= render(layout: "shared/matrix_table",
                 locals: {objects: @objects}) do |image| %>
        <div class="mt-3 mb-3">
          <%= thumbnail(image, link: url_args.merge(img_id: image.id), original: true) %>
        </div>
      <% end %>
```

Reply:
That's an easy one!  I can answer that!  You're doing <%= each %> when you should only be doing <% each %>. :)

Reply:
Agh! ❤️ You're the best! 
Had me completely flummoxed, researching the inner secrets of `yield`...

Will do a small PR to get this fixed